### PR TITLE
Centcom commanders armor change

### LINKED ISSUE #14716: Suit is the wrong sprite
The commanders suit on the bluespace artillery away mission is the wrong sprite.

### PR DESCRIPTION
**Pull request by pudl**
Fixes #14716

### CODE REVIEW

File: code/modules/awaymissions/corpse.dm [hunk 1]
```diff
@@ -425,7 +425,7 @@
 /obj/effect/mob_spawn/human/commander
 	name = "Commander"
 	uniform = /obj/item/clothing/under/rank/centcom_commander
-	suit = /obj/item/clothing/suit/armor
+	suit = /obj/item/clothing/suit/armor/centcom
```
Comment (KorPhaeron) [inline code/modules/awaymissions/corpse.dm]: This is a full suit of armour like the thunderdome. The commander is supposed to have a vest. Maybe use the bulletproof one.

/obj/item/clothing/suit/armor/bulletproof

Comment (pudl) [inline code/modules/awaymissions/corpse.dm]: hes a centcom commander so he has centcom armor

Comment (KorPhaeron) [inline code/modules/awaymissions/corpse.dm]: That's thunderdome armour though, it doesnt go with his hat at all either

Comment (pudl) [inline code/modules/awaymissions/corpse.dm]: its centcom armor

**Comment by KorPhaeron** [comment]
Fine I'll just do it

**Pull request closed without merging**